translate wp-config-sample.php

nguyenvanduocit opened the pull request.

nguyenvanduocit committed:
start translating

khoipro commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+ */
+
+// ** Thiết lập MySQL - Bạn có thể lấy các thông tin này từ host/server ** //
+/** Tên database MySQL*/
```
Thiếu 1 space sau chữ MySQL

khoipro commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+/** Tên database MySQL*/
+define('DB_NAME', 'database_name_here');
+
+/** username của database */
```
Viết hoa chữ U của Username

khoipro commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+/** Mật khẩu của database */
+define('DB_PASSWORD', 'password_here');
+
+/** hostname của database*/
```
Viết hoa chữ H của Hostname

khoipro commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+/** hostname của database*/
+define('DB_HOST', 'localhost');
+
+/** database charset sử dụng để tạo bảng database. */
```
Viết hoa chữ D của Database

khoipro commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+/**
+ * Tiền tố cho bảng database.
+ *
+ * Đặt tiền tốt cho bảng giúp bạn có thể cài nhiều site WordPress lên một database.
```
tiền `tốt`

`lên cùng` => trên cùng

nguyenvanduocit replied:
yup, em không thấy có nhiều khác biệt. mọi người nghĩ sao @htdat @dinhtungdu

khoipro replied:
Từ `trên` là thường dùng hơn `lên` đấy.
Lý do là với các hệ thống system thì `trên` là chỉ VỊ TRÍ CỦA ĐỐI TƯỢNG còn `lên` là thường chỉ TƯƠNG TÁC (ĐỘNG TỪ) với đối tượng.

Ví dụ:

`trên cùng một hệ thống`
`trên cùng một tập tin`
`trên cùng một mạng lưới` 
(động từ không cần ghi liền ngay phía trước)

Còn nói `lên` là khi:
`đẩy lên cùng một hệ thống`
`đăng lên cùng một diễn đàn`
(động từ cần bổ trợ cùng trạng ngữ thì mới đủ nghĩa bổ sung)

khoipro replied:
Mà đoạn dịch trên cũng thiếu mất chữ `cùng` mà :D

nguyenvanduocit committed:
correct the translation

khoipro commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+ * Tiền tố cho bảng database.
+ *
+ * Đặt tiền tốt cho bảng giúp bạn có thể cài nhiều site WordPress lên một database.
+ * Chỉ sử dụng số, kỹ tự và dấu gạch dưới!
```
`ký`, not "kỹ`

khoipro commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+ * Dành cho developer: Chế độ debug.
+ *
+ * Thay đổi hằng số này thành true sẽ làm hiện lên các thông báo trong quá trình phát triển.
+ * Chúng tôi mạnh mẽ đề nghị các nhà developer sử dụng WP_DEBUG trong quá trình phát triển các plugin, theme.
```
`mạnh mẽ` => khuyến cáo

khoipro commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+/** Tên database MySQL */
+define('DB_NAME', 'database_name_here');
+
+/** username của database */
```
Viết hoa chữ U của Username

nguyenvanduocit committed:
correct the translation

nguyenvanduocit committed:
correct the translation

htdat commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+ * Bạn có thể tạo ra các khóa này bằng công cụ
+ * {@link https://api.wordpress.org/secret-key/1.1/salt/ WordPress.org secret-key service}
+ * Bạn có thể thay đổi nó bất cứ lúc nào để vô hiệu hóa tất cả
+ * các cookies hiện có. Điều này sẽ buộc tất cả người dùng phải đăng nhập tại.
```
> các cookies hiện có. Điều này sẽ buộc tất cả người dùng phải đăng nhập tại.

`đăng nhập lại`

htdat commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+ * Dành cho developer: Chế độ debug.
+ *
+ * Thay đổi hằng số này thành true sẽ làm hiện lên các thông báo trong quá trình phát triển.
+ * Chúng tôi khuyến cáo các nhà developer sử dụng WP_DEBUG trong quá trình phát triển các plugin, theme.
```
> nhà developer

cái này nên là một trong hai cái dưới đây. 

1. developer 
2. nhà phát triển

>... trong quá trình phát triển các plugin, theme.

Nên bỏ `các` và thêm `và`, như thế sẽ xuôi tai hơn: 

>... trong quá trình phát triển plugin và theme.

---- 

### Tóm lại, dòng này nên chuyển 

Từ: 

```
 * Chúng tôi khuyến cáo các nhà developer sử dụng WP_DEBUG trong quá trình phát triển các plugin, theme.
``` 

Thành: 

```
 * Chúng tôi khuyến cáo các developer sử dụng WP_DEBUG trong quá trình phát triển plugin và theme.
```

htdat commented on dist/wp-config-sample.php:
```diff
@@ -0,0 +1,91 @@
+ */
+define('WP_DEBUG', false);
+
+/* Đó là tất cả thiết lập, ngưng sửa từ phần này trở xuống, Chúc bạn viết blog vui vẻ. */
```
```
/* Đó là tất cả thiết lập, ngưng sửa từ phần này trở xuống, Chúc bạn viết blog vui vẻ. */
```

Trước `Chúc` là dấu chấm (.), sửa thành: 

```
/* Đó là tất cả thiết lập, ngưng sửa từ phần này trở xuống. Chúc bạn viết blog vui vẻ. */
```

nguyenvanduocit committed:
correct the translation

htdat committed:
Update thêm một số thay đổi nhỏ

htdat committed:
Sửa lại chút để cho số line giống với ở file gốc

89 lines ở đây https://github.com/WordPress/WordPress/blob/4.9/wp-config-sample.php

nguyenvanduocit commented:
Cảm ơn anh Đạt fix giùm em mấy chỗ. Ngon thì merge nhé. Lần này squash trước, lần trước quên phải fix lại.
